UnifiedCASManagement/CHANGELOG.md

```diff
@@ -16,10 +16,14 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [2.0.0] - 2023-05-26
```

Comment:
Did we intend to do a major version upgrade since that indicates breaking changes. Can you call out the breaking API changes?